website/docs/docs/cloud/manage-access/set-up-sso-google-workspace.md

```diff
@@ -49,7 +49,7 @@ Client Secret for use in dbt Cloud.
 | **Application type**   | internal     | required |
 | **Application name**   | dbt Cloud    | required |
 | **Application logo**   | Download the logo <a href="https://www.getdbt.com/ui/img/dbt-icon.png" target="_blank" rel="noopener noreferrer">here</a> | optional |
-| **Authorized domains** | `getdbt.com` (US) `dbt.com` (EMEA or AU) | If deploying into a VPC, use the domain for your deployment |
+| **Authorized domains** | `getdbt.com` (US multi-tenant) `getdbt.com` and `dbt.com`(US Cell 1) `dbt.com` (EMEA or AU) | If deploying into a VPC, use the domain for your deployment |
```

Comment:
not sure why there are two `getdbt.com` entries here so sugg removing one and consolidating the `dbt.com` one:

```suggestion
| **Authorized domains** | `getdbt.com` (NA multi-tenant) and `dbt.com` (US Cell 1, EMEA, or AU) | If deploying into a VPC, use the domain for your deployment |
```

Reply:
Customers will need to add both if they're on the cell 1 infrastructure